helmctlr/helm.go

```diff
@@ -122,23 +124,39 @@ func (c Controller) installOrUpdate(r *unstructured.Unstructured) error {
 	}
 
 	rlsName := c.releaseName(r)
+	// TODO: should we set configmap owner references before install/update, in
```

Comment:
Doesn't that depend on when the configmap gets created? Do we know how to query for it before the release is made?

to be safe, I would think we want to do:

```
- check if configmap exists
   - if so, make sure it has ownerrefs
- helm update/install
- check if configmap exists
   - if so, make sure it has ownerrefs
```

Reply:
We can basically query the same way both times, it's pretty easy, we would be effectively listing configmaps before we run the release to see if any previous release CMs never got their owner refs set (ie, in case we fail multiple times in a row during/after install/upgrade).

Reply:
I've updated to have the code do the check/update before, and after install/upgrade.